addon_service/oauth1/views.py

```diff
@@ -0,0 +1,42 @@
+from http import HTTPStatus
+
+from asgiref.sync import async_to_sync
+from django.http import HttpResponse
+
+from addon_service.authorized_storage_account.models import AuthorizedStorageAccount
+from addon_service.common.known_imps import AddonImpNumbers
+from addon_service.oauth1.utils import get_access_token
+from addon_service.oauth_utlis import update_external_account_ids
+
+
+def oauth1_callback_view(request):
+    oauth_token = request.GET["oauth_token"]
+    oauth_verifier = request.GET["oauth_verifier"]
+
+    pk = request.session.get("oauth1a_account_id")
+    del request.session["oauth1a_account_id"]
+
+    account = AuthorizedStorageAccount.objects.get(pk=pk)
+
+    oauth1_client_config = account.external_service.oauth1_client_config
+    final_credentials, other_info = async_to_sync(get_access_token)(
+        oauth1_client_config.access_token_url,
+        oauth1_client_config.client_key,
+        oauth1_client_config.client_secret,
+        oauth_token,
+        account.credentials.oauth_token_secret,
+        oauth_verifier,
```

Comment:
nit: kwargs preferred to positional args, especially with this many arguments

Reply:
Yep, but also they are kinda self-explainatory